tests/framework/TestResources.php

```diff
@@ -52,6 +52,7 @@ class TestResources
     const VALUE2        = 'value2';
     const VALUE3        = 'value3';
     const ACCOUNT_NAME  = 'myaccount';
+    const SAS_TOKEN     = 'st=2016-12-01T19%3A43%3A00Z&se=2016-12-02T19%3A43%3A00Z&sp=rwdl&sv=2015-12-11&sr=c&sig=aGVsbG8gd29ybGQ%3D';
```

Comment:
@katmsft We just provide a sample SAS token in tests because a SAS token would be a user-provided piece of information, just like an account name or an account key. The use case for this feature is not for a user to provide an account name/key and that we generate a SAS token and that we then use that token to authenticate, but instead for the user to provide a SAS token directly that we use to authenticate their requests.
Also, adding SAS token generation seems out of scope as that's not needed to implement SAS authentication. However, I do have some PHP code for that as well that I wrote a while ago and I can open another PR for that.